pypi_count/py_counter.py

```diff
@@ -31,6 +31,13 @@ def count_if_statements(self):
         """Counting the if statements."""
         return len(m.findall(self.search, m.If()))
 
+    def count_while(self):
```

Comment:
The names of these methods should match the style of other methods: they should end in `_statements` if the construct they are counting is a statement, or the corresponding word if they are not a statement (for instance, in `count_function_definitions`, "definition" replaced "statement".

Reply:
@solisa986

Reply:
@rachaelharris